.gitlab-ci.yml

```diff
@@ -90,13 +90,16 @@ check-std:
   stage:                           check-workspace
   <<:                              *docker-env
   script:
-    - cargo check --verbose --all-features --release --all
+    - all_crates=("core" "alloc" "utils" "lang2" "lang2/macro" "cli")
```

Comment:
Can we somehow fix this code dupe? Updating this would be really daunting.

Reply:
I'd loop all dirs, but there you have `model` and `lang` which do not need tests.
Will think how to make it better.

Reply:
Created `${ALL_CRATES}` and `${WASM_CRATES}` global variables with lists of the crates.